pkgs/applications/editors/vscode/extensions/vadimcn.vscode-lldb/default.nix

```
@@ -64,6 +72,9 @@ in stdenv.mkDerivation {
 
   postConfigure = ''
     cp -r ${nodeDeps}/lib/{node_modules,package-lock.json} .
+  '' + lib.optionalString stdenv.isDarwin ''
+    mkdir -p "$(pwd)/home"
+    export HOME="$(pwd)/home"
```

```suggestion
    export HOME="$TMPDIR/home"
    mdkir $HOME
```
we shouldn't just put home into $PWD because it could easily influence the files copied to $out